cmd/lakectl/cmd/local.go

```diff
@@ -54,7 +54,7 @@ func getLocalSyncFlags(cmd *cobra.Command) syncFlags {
 func getLocalArgs(args []string, requireRemote bool, considerGitRoot bool) (remote *uri.URI, localPath string) {
 	idx := 0
 	if requireRemote {
-		remote = MustParseRefURI("path", args[0])
+		remote = MustParseRefWithPathURI("path", args[0])
```

Comment:
Why not use `MustParsePathURI`? it will verify that the lakefs ref is to a valid path

Reply:
`MustParsePathURI` requires a fully qualified name where path != nil.
In our case ref can be anything and path can exist or not